docker-compose.override-macos.yml

```diff
@@ -0,0 +1,12 @@
+## override file for mac
+
+version: '3'
+services:
+  client:
+    command: ["bash", "-c", "yarn --network-timeout 1000000000 && yarn run start --host 0.0.0.0"]
+    volumes:
+      - ./client/:/client/:delegated
+      - client_node_modules:/client/node_modules
```

Comment:
Mapping the node_modules directory can be tricky as some packages rely on libs in the underlying platform (`bcrypt` for example). This can cause errors if the container and the os used to run the container are different.

Reply:
so this actually makes the container not map between host and container. it creates a dummy volume that should get cached

Reply:
I understand this better now. Volume is being created and delegated not mapped from the host machines file system.